src/authentication/protected-routes.handler.ts

```diff
@@ -10,31 +10,20 @@ export const withProtectedRoutesHandler = (
   const { rootPath, loginPath, logoutPath } = admin.options;
 
   router.use((req, res, next) => {
-    if (isAdminAsset(req.originalUrl)) {
-      next();
-    } else if (
-      req.session.adminUser ||
-      // these routes doesn't need authentication
+    if (
+      // these routes don't need authentication
```

Comment:
[OPTIONAL] 

- create method `doesNeedAuthentication()`
- then `if (!doesNeedAuthentication(req.originalUrl.)`
- remove comment

Thanks to this extra comment wouldn't be needed and code would be easier to read.